client/rpc_test.go

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package client_test
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	plogrus "perun.network/go-perun/log/logrus"
+	pkgtest "perun.network/go-perun/pkg/test"
+
+	"github.com/perun-network/erdstall/client"
+	"github.com/perun-network/erdstall/operator"
+	optest "github.com/perun-network/erdstall/operator/test"
+	ttest "github.com/perun-network/erdstall/tee/test"
+)
+
+var shortWait = 300 * time.Millisecond
+var longWait = 10 * time.Second
+
+// TestRPC_ClientOp is a test which tests all functions of the Operator
+// over a websocket connection.
+//
+// It uses a mocked remote enclave to simulate failing functions.
+// The construction looks a bit like this:
+// Client <-> RPC <-> MockedRPCOperator <-> OP
+func TestRPC_ClientOp(t *testing.T) {
+	plogrus.Set(logrus.TraceLevel, &logrus.TextFormatter{ForceColors: true})
```

Comment:
do we really always want trace logging in ci tests? I'd rather set this to `Info` and only use trace during debugging.

Reply:
Removed it.